content/articles/creating-an-application-using-beeware/index.md

```diff
@@ -0,0 +1,259 @@
+        button1.style.padding_left = 10
+
+        button2 = toga.Button('2', on_press=partial(self.enterdata, number='2'))
+
+        button3 = toga.Button('3', on_press=partial(self.enterdata, number='3'))
+
+        buttonmultiply = toga.Button('×', on_press=partial(self.enterdata, number='*'))
+
+        buttondot = toga.Button('.', on_press=partial(self.enterdata, number='.'))
+        buttondot.style.padding_left = 10
+
+        button0 = toga.Button('0', on_press=partial(self.enterdata, number='0'))
+
+        buttonclear = toga.Button('C', on_press=partial(self.enterdata, number='C'))
+
+        buttondivide = toga.Button('÷', on_press=partial(self.enterdata, number='/'))
+
+        calculate = toga.Button('CALCULATE', on_press=self.calculate)
+        calculate.style.width = 150
+        calculate.style.padding_top = 30
+        calculate.style.padding_left = 30
+
+        # adding
+        box1.add(self.input_text)
+
+        box2.add(calculate)
+
+        box3.add(button7)
+        box3.add(button8)
+        box3.add(button9)
+        box3.add(buttonplus)
+
+        box4.add(button4)
+        box4.add(button5)
+        box4.add(button6)
+        box4.add(buttonminus)
+
+        box5.add(button1)
+        box5.add(button2)
+        box5.add(button3)
+        box5.add(buttonmultiply)
+
+        box6.add(buttondot)
+        box6.add(button0)
+        box6.add(buttonclear)
+        box6.add(buttondivide)
+
+        # adding in main box
+        main.add(box1)
+        main.add(box2)
+        main.add(box3)
+        main.add(box4)
+        main.add(box5)
+        main.add(box6)
+
+        main.style.update(direction=COLUMN)
+
+        self.main_window = toga.MainWindow(title=self.formal_name)
+        self.main_window.content = main
+        self.main_window.show()
+
+    def enterdata(self, widget, number):
+        if (number == "C"):
+            self.input_text.value = ""
+        else:
+            self.input_text.value = self.input_text.value + number
+
+    def calculate(self, widget):
+        output = eval(self.input_text.value)
+        self.input_text.value = output
+
+
+def main():
+    return SimpleCalculator()
+
+
+```
+We have defined all the necessary buttons required to make our application. Each button is a toga button with some having a little padding to the top or left or both. There is a callback method `on_press` defined within the same buttons. We make use of a class defined function `enterdata` that takes a number or an operand such as `+`, `-`, `*`, `÷`, `.` and `C`. and appends it to the input value.
+
+We make use of the `partial` built in function in python that takes a function and a some inputs pre-filled to return a fully-filled function. Our partial function takes our enterdata function and a number value to be passed as final parameter to the function.There is also a `CALCULATE` button that when pressed 
+calls the `calculate` function.
```

Comment:
Function names and variables should be enclosed in backticks. Do that across the entire article.